components/admin_console/license_settings/enterprise_edition/enterprise_edition_left_panel.tsx

```diff
@@ -0,0 +1,214 @@
+
+    return (
+        <div className='EnterpriseEditionLeftPanel'>
+            <div className='title'>
+                {`Mattermost ${skuName}`}{freeTrialBadge(isTrialLicense)}
+            </div>
+            <div className='subtitle'>
+                <FormattedMessage
+                    id='admin.license.enterpriseEdition.subtitle'
+                    defaultMessage='This is an Enterprise Edition for the Mattermost {skuName} plan'
+                    values={{skuName}}
+                />
+            </div>
+            <div className='licenseInformation'>
+                <div className='license-details-top'>
+                    <span className='title'>{'License details'}</span>
+                    {(expirationDays <= 30) &&
+                        <span className='expiration-days'>
+                            {`Expires in ${expirationDays} day${expirationDays > 1 ? 's' : ''}`}
+                        </span>
+                    }
+                </div>
+                {
+                    renderLicenseContent(
+                        license,
+                        isTrialLicense,
+                        issued,
+                        startsAt,
+                        expiresAt,
+                        handleRemove,
+                        isDisabled,
+                        removing,
+                    )
+                }
+            </div>
+            <div className='license-notices'>
+                {upgradedFromTE ? <>
+                    <p>
+                        {'When using Mattermost Enterprise Edition, the software is offered under a commercial license. See '}
+                        <a
+                            role='button'
+                            onClick={openEELicenseModal}
+                            className='openEELicenseModal'
+                        >
+                            {'here'}
+                        </a>
+                        {' for “Enterprise Edition License” for details. '}
+                        {'See NOTICE.txt for information about open source software used in the system.'}
+                    </p>
+                </> : <p>
+                    {'This software is offered under a commercial license.\n\nSee ENTERPRISE-EDITION-LICENSE.txt in your root install directory for details. See NOTICE.txt for information about open source software used in this system.'}
+                </p>
+                }
+            </div>
+        </div>
+    );
+};
+
+const renderLicenseContent = (
+    license: ClientLicense,
+    isTrialLicense: boolean,
+    issued: JSX.Element,
+    startsAt: JSX.Element,
+    expiresAt: JSX.Element,
+    handleRemove: (e: any) => Promise<void>,
+    isDisabled: boolean,
+    removing: boolean,
+) => {
+    // Note: DO NOT LOCALISE THESE STRINGS. Legally we can not since the license is in English.
+    let skuName = license.SkuShortName;
+    if (isTrialLicense) {
+        skuName = `${license.SkuShortName} License Trial`;
```

Comment:
Also here we should use `skuName` so that it doesn't show lower case the name `enterprise` 
<img width="569" alt="Screen Shot 2021-11-16 at 12 40 20 PM" src="https://user-images.githubusercontent.com/936315/142037300-3c3ea987-66cf-47e4-8f02-0b73b7a137be.png">